### DISCUSSION TITLE
i cannot use discord :(

### DISCUSSION THREAD

**Discussion by brah-mcdude** [akkadotnet/akka.net#6352] [General]
posted also in [gitter](https://gitter.im/akkadotnet/akka.net): Brah McDude @brah-mcdude [15:10](https://gitter.im/akkadotnet/akka.net?at=63c5af267de82d26162fa16b)
what should i do if i don't have a phone number? i cannot use discord!!! i need to keep my anonymity. where can i ask questions? can we use the "discussions" tab in github?

**Comment by Aaronontheweb** [comment]
@brah-mcdude you can absolutely ask questions here! And these also get cross-posted into Discord too.
